Vagrantfile

```diff
@@ -89,7 +89,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.define "database" do |database|
     database.vm.hostname = "database"
-    database.hostmanager.aliases = %w(database.service.ashlar.internal)
+    database.hostmanager.aliases = %w(database.service.driver.internal)
```

Comment:
theres a hostmanager config for `app.service.ashlar.internal` in this file, might want to switch that one to driver